### PR TITLE
Update quickstart for 3.1.0

### PR DESCRIPTION
**Pull request by jonchiu**
* Revise console consumer/producer example
* Add new alerts walkthrough
* Update screenshots

### CODE REVIEW

**Comment by jonchiu** [comment]
@theduderog

File: docs/quickstart.rst [hunk 1]
```diff
@@ -263,9 +276,194 @@ Now that we have all of the Docker dependencies installed, we can create a Docke
...
+To begin receiving alerts on anomalous events in your monitoring data, let's
+create a trigger based on some criteria. Click the "Triggers" navigation item and
+then select "+ New trigger".
+
+Let's configure a trigger to fire when we the difference between our actual
+consumption and expected consumption is greater
+than 1000 messages over a 5 second interval. This means as we are producing
+10,000 messages every 10 seconds, we should expect our consumers to be able
+to keep up with our producers to within 1000 messages over a 5 second
+interval. Let's fill out our form:
```
Comment (theduderog) [inline docs/quickstart.rst]: This a kinda hard to follow

**Comment by theduderog** [comment]
Thanks, @jonchiu!  A few suggestions and LGTM

**Comment by theduderog** [comment]
LGTM
